### ISSUE TITLE
Value-list disabled prop on render function

### ISSUE THREAD

**Issue by kat10140** [Esri/calcite-app-components#424]
As it is now-
```
  render() {
    const { dataForFilter, handleFilter, disabled, filterEnabled, loading } = this;
    return (
      <Host>
        <div class={CSS.container} aria-busy={loading}> //HERE 
          <header>
            {filterEnabled ? (
              <calcite-filter
                data={dataForFilter}
                textPlaceholder={TEXT.filterPlaceholder}
                aria-label={TEXT.filterPlaceholder}
                onCalciteFilterChange={handleFilter}
              />
            ) : null}
            <slot name="menu-actions" />
          </header>
          <slot />
          {this.renderScrim()}
        </div>
      </Host>
    );
  }
``` 

Not having disabled prop on the div in the render is throwing an error 

ref: #405 

also, pick-list render has `aria-disabled=disabled` on the render, and Franco says should be `disabled={disabled}` 😬

**Comment by kat10140** [comment]
guessing the `aria-disabled={disabled}` thing didn't need a fix. Verified.
